### ISSUE TITLE
ng lint not working after upgrade to beta 31 (from beta 24)

### ISSUE THREAD

**Issue by emcpadden** [angular/angular-cli#4748]
After upgrading to angular-cli beta 31 (from beta 24) the ng lint np longer works.

### OS?
Mac OSX Sierra

### Versions.  
node --version: 6.9.1
ng --version:
@angular/cli: 1.0.0-beta.31
node: 6.9.1
os: darwin x64
@angular/common: 2.4.7
@angular/compiler: 2.4.7
@angular/core: 2.4.7
@angular/forms: 2.4.7
@angular/http: 2.4.7
@angular/material: 2.0.0-alpha.10
@angular/platform-browser: 2.4.7
@angular/platform-browser-dynamic: 2.4.7
@angular/router: 3.4.7
@angular/cli: 1.0.0-beta.31
@angular/compiler-cli: 2.4.7

### Repro steps.
I upgraded to beta 31 of the angular beta 24 version of angular-cli following the upgrade instructions.  The last step in the instructions said to do `ng upgrade` which I did and it only asked about overwriting the readme.

Since it did not update the angular modules, I updated them in package.json and killed the node_modules folder and re-did the node install.

I got everything working but when I try to do a `ng lint` it says the following:
`No lint config(s) found. If this is not intended, run "ng update".`

Is there something I need to do to get the ng lint to work in beta 31?

Thanks,
... Ed

**Comment by clydin** [comment]
add the following to `angular-cli.json` after the `e2e` section (adjusting for your path setup):
```
  "lint": [
    {
      "files": "src/**/*.ts",
      "project": "src/tsconfig.json"
    },
    {
      "files": "e2e/**/*.ts",
      "project": "e2e/tsconfig.json"
    }
  ],
```

**Comment by emcpadden** [comment]
@clydin Thanks again!!! You have saved me twice in 24 hours 👍

**Comment by killrazor** [comment]
Verifying that @clydin's fix has corrected the same issue but when upgrading from 1.0.0-beta.25.5 to 1.0.0-rc.4.
